can-construct.js

```diff
@@ -675,8 +755,9 @@ assign(Construct, {
  *
  * @param {*} args The arguments passed to the constructor.
  *
- * @return {Array|undefined} If an array is returned, the array's items are passed as
- * arguments to [can-construct::init init]. The following example always makes
+ * @return {Array|undefined|can-construct.ReturnValue} If an array is returned, the array's items are passed as
+ * arguments to [can-construct::init init]. If [can-construct.ReturnValue] is returned then the value that passed to it
+ * will be used as a value for a new instance. The following example always makes
```

Comment:
Change:

> If [can-construct.ReturnValue] is returned then the value that passed to it
> will be used as a value for a new instance.

To something more like:

If a [can-construct.ReturnValue] instance is returned, the `ReturnValue` instance's 
value will be returned as the result of calling `new Constructor()`.